numpy/lib/arrayterator.py

```diff
@@ -80,9 +80,8 @@ class Arrayterator(object):
 
     >>> for subarr in a_itor:
     ...     if not subarr.all():
-    ...         print(subarr, subarr.shape)
-    ...
-    [[[[0 1]]]] (1, 1, 1, 2)
+    ...         print(subarr, subarr.shape) # doctest: +SKIP
+    >>> # [[[[0 1]]]] (1, 1, 1, 2)
```

Comment:
Why are we skipping this?

Reply:
Again, we can't `print()` ndarrays with refguide -- it will expect `random` for whatever reason--either we use the repr or add the skip directive. Maybe @ev-br knows more.